MvvmCross.Android.Support/V7.AppCompat/MvxAppCompatViewPresenter.cs

```diff
@@ -153,6 +154,46 @@ public override MvxBasePresentationAttribute CreatePresentationAttribute(Type vi
             return base.CreatePresentationAttribute(viewModelType, viewType);
         }
 
+        public override Task<bool> ChangePresentation(MvxPresentationHint hint)
+        {
+            if (hint is MvxPagePresentationHint pagePresentationHint)
+            {
+                var request = new MvxViewModelRequest(pagePresentationHint.ViewModel);
+                var attribute = GetPresentationAttribute(request);
+
+                if (attribute is MvxViewPagerFragmentPresentationAttribute pagerFragmentAttribute)
+                {
+                    ViewPager viewPager = null;
+
+                    // check for a ViewPager inside a Fragment
+                    if (pagerFragmentAttribute.FragmentHostViewType != null)
+                    {
+                        var fragment = GetFragmentByViewType(pagerFragmentAttribute.FragmentHostViewType);
+                        viewPager = fragment.View.FindViewById<ViewPager>(pagerFragmentAttribute.ViewPagerResourceId);
+                    }
+
+                    // check for a ViewPager inside an Activity
+                    if (pagerFragmentAttribute.ActivityHostViewModelType != null)
```

Comment:
Shouldn't this be an `else if` otherwise it would overwrite the first found viewPager if both are present.

Reply:
True. @markuspalme can you please change this also when navigating forward to a ViewPager fragment? Looks like an easy change :)